app/src/components/template/Dashboard.js

```diff
@@ -11,20 +11,32 @@ import {
 import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
 import MenuIcon from '@material-ui/icons/Menu';
 import { withStyles } from '@material-ui/core/styles';
+import MenuItem from '@material-ui/core/MenuItem';
 import { mainListItems } from './listItems';
 import { Styles as OBStyles } from '../../styles/OrderBookSnapshot';
 import OrderBookSnapshot from '../OrderBookSnapshot';
 import Home from '../Home';
 import NotFound from '../NotFound';
 import { styles } from '../../styles/Dashboard';
+import OrderBookService from '../../services/OrderBookService';
 
 class Dashboard extends Component {
     constructor(props) {
         super(props);
 
         this.state = {
             open: false,
+            selectedInstrument: '',
+            instruments: [],
         };
+
+        const { instruments } = this.state;
+
+        OrderBookService.getInstrumentsList().then(response => {
```

Comment:
Shouldn't this be in `componentDidMount`?

Reply:
👍